app/views/patients/show.html.erb

```diff
@@ -11,6 +11,10 @@
           <div class="is-grouped">
             <%= link_to 'Edit', edit_patient_path(@patient), class: 'button is-info is-light' %> 
             <%= link_to 'Download Health Card', patient_health_card_path(@patient, format: 'smart-health-card'), class: 'button is-info is-light' %>
+            <%= form_with url: patient_health_card_path(@patient, format: 'pdf'), method: :post do |f| %>        
```

Comment:
```suggestion
            <%= form_with url: patient_health_card_path(@patient, format: 'pdf'), class: 'button is-info is-light', method: :post do |f| %>        
```

This should fix the spacing issue:

![Screen Shot 2021-05-14 at 1 17 21 PM](https://user-images.githubusercontent.com/41651655/118305998-c198db80-b4b6-11eb-9a7e-23fa94f0c416.png)

Reply:
Thanks!